### ISSUE TITLE
Overfitting after data augmentation?

### ISSUE THREAD

**Issue by hjzhang1018** [deepchem/deepchem#2159]
To unbalance my dataset, I used Smiles Augmentation method to increase the positive compound numbers (created around 10 times more positive smiles). Then I used this augmented dataset (Positive/Negative ratio is around 1.5) for later training. I used Random Forest and GraphConvModel, they both get "great" scores, almost all evaluation scores are over 0.98, in Random Forest, the roc_auc_score and recall score is 1.
I think what really happens here may be overfitting. But I only trained my data for 10 epochs.

I wonder if there's any way to fix this? Is this because I used wrong augmentation method? 

Thanks for any help!

**Comment by alat-rights** [comment]
Could you please confirm overfitting by computing metrics with an independent test set?

**Comment by hjzhang1018** [comment]
@alat-rights Thank you, good point! I'll find an independent test set and see how good the scores are.

**Comment by alat-rights** [comment]
Hey, were you able to resolve the issue?
